server/routes/dev.py

```diff
@@ -49,3 +49,9 @@ def disaster_dashboard():
       dc.get_places_in(["europe"], "Country").get("europe", []))
   return flask.render_template('dev/disaster_dashboard.html',
                                european_countries=european_countries)
+
+@bp.route('/event-page')
```

Comment:
nit: "event" rather than "event-page" since they're all pages. drop the "-page" for the route and filenames too

Reply:
Renamed all `event-page` instances to `event`